airbyte-workers/src/main/java/io/airbyte/workers/normalization/NormalizationRunnerFactory.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Airbyte
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.workers.normalization;
+
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.workers.normalization.NormalizationRunner.NoOpNormalizationRunner;
+import io.airbyte.workers.process.ProcessBuilderFactory;
+import java.util.Map;
+
+public class NormalizationRunnerFactory {
+
+  private static final Map<String, DefaultNormalizationRunner.DestinationType> NORMALIZATION_MAPPING =
+      ImmutableMap.<String, DefaultNormalizationRunner.DestinationType>builder()
+          .put("airbyte/destination-bigquery", DefaultNormalizationRunner.DestinationType.BIGQUERY)
```

Comment:
What if instead of using the image name you use the JsonConfig and test for the presence of the well known key? this way you can hide all the logic for deciding why normalizer to use in the factory.

Reply:
to make sure i understand your suggestion, you mean adding a key like `normalizer_destination_type: "bigquery"` to the config? i'm fine with that.